### DISCUSSION TITLE
Synchronize several installations on different machines

### DISCUSSION THREAD

**Discussion by Erl-gry** [OrcaSlicer/OrcaSlicer#856] [Q&A]
Hi!

I have installed Orca slicer on different Windows computers. I have noticed that the history of the projects is different on each computer, of course depending on which project was processed on which computer.
 
Is it possible to synchronize the respective installations with each other?

To have the already created projects (3mf files) available on each computer, I use Syncthing (similar to Dropbox only self-hosted). Would it be possible to synchronize the configurations of OrcaSlicer as well, so that everything is identical on each computer?
Is it enough to synchronize the directory C:\Users\USER\AppData\Local\OrcaSlicer?

Any help would be appreciated :)

**Comment by SoftFever** [comment]
@Frontholz it should be the c:\Users\USER\AppData\Roaming\OrcaSlicer\ folder

**Reply by SoftFever** [reply]
More specifically, I think project history is stored in OrcaSlicer.conf file. But I didn't double-check it.

**Reply by carlosjln** [reply]
This comment saved me from having to reload 30+ projects to restore my work history and make it easy to access 🙏 thanks!

**Comment by julie777** [comment]
@Frontholz I am not sure what is in Appdata/local vs appdata/roaming. They are not always used correctly when software is designed for multiple platforms. Usually any sharable information for Windows should be in appdata/roaming and only cache or machine specific information should be in appdata/local.

The design of Windows is intended to be able to share all off appdata/roaming to any machine that you log in to, but very few machines are set up that way.

If what you want is to have everything the same on multiple machines, that are not in use simultaneously, then your soluton of using a mounted directory, such as dropbox for appdata/roaming/orca should work. In addition if you want all your projects to be accessible then a shared folder for projects would handle that.

**Comment by julianjc84** [comment]
EDIT solved: https://github.com/SoftFever/OrcaSlicer/pull/6780
add a folder called "data_dir" inside the portable app does what i want.

Would like to learn more about managing orca slicer in regards to.
Running multiple version of orca slicer. Stable and DevBuilds, Do they all save to the same location?
C:\Users\USERNAME\AppData\Roaming\OrcaSlicer
How does it handle conflicting settings from updated features as your roll forward and backward between different orca builds.
How to run different orca versions to handle multiple printers. Example if you have multiple different printers that you use. to cleanly separate out the settings.
How to sync these setting between multiple computers in the house, Desktop/Laptop.

I come from superslicer background and you can create a folder called "configurations" inside the app folder, and it wont use the windows default "appdata" folder to store settings. 
This enables you to separate out different "printers" and "configuration settings".
Run multiple versions simultaneously (stable/dev/past builds) and not have conflicting config files that don't understand older features/settings and you can even store the app on a network NAS / dropbox / one drive / Google Drive to sync between multiple computers.

i am very keen to get into Orca.
